src/core_modules/capture-ui/PreviewImage/PreviewImage.component.js

```diff
@@ -0,0 +1,66 @@
+// @flow
+import React from 'react';
+import { withStyles } from '@material-ui/core/styles';
+
+const styles = () => ({
+    container: {
+        position: 'relative',
+        display: 'inline-block',
+        '&:hover $image': {
+            opacity: 0.5,
+        },
+        '&:hover $icon': {
+            visibility: 'visible',
+        },
+    },
+    image: {
+        height: 80,
+        maxWidth: 120,
+        objectFit: 'contain',
+    },
+    icon: {
+        position: 'absolute',
+        padding: '4px 4px 2px 4px',
+        left: '50%',
+        bottom: '50%',
+        transform: 'translate(-50%, 50%)',
+        background: 'rgba(255, 255, 255, 0.5)',
+        cursor: 'pointer',
+        visibility: 'hidden',
+    },
+});
+
+const PreviewImagePlain = (props: {
+    url: string,
+    classes: {
+        container: string,
+        image: string,
+        icon: string,
+    },
+}) => {
+    const { url, classes } = props;
+
+    return (
+        <div className={classes.container}>
+            <a
+                href={url}
```

Comment:
Hey, clicking on the image should open the full image in a new tab. To achieve this, when creating the link `href`, you can remove `dimension=small` from this URL. WDTY?